storybook/pages/StatusDialogPage.qml

```diff
@@ -37,11 +36,8 @@ SplitView {
         StatusDialog {
             id: dialog
 
-            closePolicy: Popup.NoAutoClose
```

Comment:
It was impossible to close the popup with a custom header; had to restart StoryBook :)